IFC4x3/Properties/i/IsAJointControlEquipment_1y8uHbp6z5vfRtA_1nm6D8/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates whether a security equipment checks mechanical functionality (RFI functionality) or not.
```

Comment:
A property of something else (security equipment)?

Reply:
No It's indicate if the equipement mechanical functionnality is checked by an other security system. I modified the description. Commit : c8245c45a7f488800e3e5c5dd32f498fffa28154